src/app/pages/cart/components/cart-summary/cart-summary.component.ts

```diff
@@ -36,9 +50,41 @@ export class CartSummaryComponent implements OnInit {
         this.subTotal = totalPrice;
         this.cdr.detectChanges();
       });
+
+    this.cartFacadeService.getCart()
+      .pipe(takeUntilDestroyed(this.destroyRef))
+      .subscribe((cart: ICartItem[]) => this.cart = cart);
   }
 
   buildTranslationKey(relativeKey: string): string {
     return `cart.${relativeKey}`;
   }
+
+  checkout(): void {
+    const newOrder: IOrder = {
+      products: this.cart,
```

Comment:
if you want to get the `cart` only for this place, you can use it here directly as async operation